about.html

```diff
@@ -0,0 +1,122 @@
+            </button>
+            <ul id="menu" class="menu-inactive">
+                <li>
+                    <a class="nav-item" href="./index.html" tabindex="3">Home</a>
+                </li>
+                <li>
+                    <a class="nav-item" href="#" tabindex="4">Agenda</a>
+                </li>
+                <li>
+                    <a class="nav-item" href="#" tabindex="5">Join</a>
+                </li>
+                <li>
+                    <a class="nav-item" href="#" tabindex="6">Contact</a>
+                </li>
+            </ul>
+        </div>
+        </nav>
+    <section class="contact" id="contact" tabindex="7">
+        <div class="contact-text">
+
+            <h1 aria-label="Annual International Science Research Conference 2023" class="text-bg fw-bold p-1">
+                Annual International
+                <br />
+                Science Research Conference 2023
+            </h1>
+            <hr class="about-hr" />
+            <p aria-label="The International Conference brings together the community of experts, academics, and activists who 
+            comprise but not limited, people support LGBTQIA+ group and different races every year.
+            This year, we hope to expand our invitation list including organizations and individuals who want to 
+            work with us on shared research, free culture, common respect, open knowledge, without borders!">
+                The International Conference brings together the community of experts, academics, and activists who
+                comprise but not limited, people support LGBTQIA+ group and different races every year.
+                This year, we hope to expand our invitation list including organizations and individuals who want to
+                work with us on shared research, free culture, common respect, open knowledge, without borders!
+            </p>
+            <p id="address">
+                Please contact us through Email for any further questions about AISRC 2023
+            </p>
+            <a>aisrc@question.org</a>
+        </div>
+
+    </section>
+
+    <section class="logo2023" id="logo">
+        <div class="brief">
+            <div class="about-text">
+                <h1 aria-label="About me">AISRC 2023 Logo</h1>
+                <hr class="about-hr" />
+                <p aria-label="The logo of AISRC 2023 was decided through the logo competition 
+                from 8. June to 7 July.">The logo of AISRC 2023 was decided through the logo competition
+                    from 8. June to 7 July.
+                </p>
+            </div>
+        </div>
+        <div id="logo-container">
+            <img src="./img/logo3.png" alt="AISRC Logo" />
+            <div id="footer-info-text">
+                <h1>AISRC</h1>
+                <p>Annual International Science Research Conference</p>
+            </div>
+
+        </div>
+        <hr id="seperator" />
```

Comment:
- [ ] The divider appears very long unlike what we have on Behance design, kindly fix it.
- [ ] The one I highlighted with yellow is also bigger in width than the Behance design, kindly fix it.

## Your design
![image](https://user-images.githubusercontent.com/86200356/232961307-026e2e21-36b8-480e-a479-efc8a541eac8.png)


## Behance design
![image](https://user-images.githubusercontent.com/86200356/232961383-b1e554b3-92fd-4d8b-ac3d-24d91580d051.png)